faktory/_proto.py

```diff
@@ -167,11 +167,12 @@ def fetch(self, queues) -> Optional[dict]:
     def reply(self, cmd, data=None):
         if self.debug: self.log.debug("< {} {}".format(cmd, data or ""))
         s = cmd
-        if data is not None:
-            if type(data) is dict:
-                s = "{} {}".format(s, json.dumps(data))
-            else:
-                s = "{} {}".format(s, data)
+        if not data:
+            return
```

Comment:
If you return here, then self.reply() without any Argos wont get sent to the server?

Reply:
Argos = args